src/bin/replicator.rs

```diff
@@ -94,7 +95,14 @@ fn main() {
     // TODO: ask network what slice we should store
     let entry_height = 0;
 
+    // Create the RocksDb ledger, eventually will simply repurpose the input
+    // ledger path as the RocksDb ledger path
+    let db_ledger_path = format!("{}/{}", ledger_path.unwrap(), DB_LEDGER_DIRECTORY);
+    let db_ledger = Arc::new(RwLock::new(
+        DbLedger::open(&db_ledger_path).expect("Expected to be able to open database ledger"),
+    ));
     let (replicator, leader_info) = Replicator::new(
+        db_ledger,
```

Comment:
I think it'd be better if you passed the ledger_path as the argument, did the opening in src/fullnode.rs.  This way we can move to sled without changing this code.

Reply:
I think this comment was meant for bin/fullnode.rs? I made the change there :)

Reply:
is the change not appropriate here, as well?

Reply:
The replicator doesn't open a fullnode, but I can move the open logic to src/replicator.rs to parallel the change in src/fullnodes.rs